lambdas/db-migration/src/migrations/20201014105016_create_providers_table.ts

```diff
@@ -12,6 +12,7 @@ export const up = async (knex: Knex): Promise<void> =>
     table
       .enum('protocol', ['http', 'https', 'ftp', 'sftp', 's3'])
       .comment('Protocol for the provider')
+      .defaultTo('http')
```

Comment:
This is required as api/models/schemas/pdr has:

    protocol: {
      title: 'Protocol',
      type: 'string',
      enum: ['http', 'https', 'ftp', 'sftp', 's3'],
      default: 'http',
    },

and the model is creating this default by...er...default.